docs/hub/spaces-run-with-docker.md

```diff
@@ -0,0 +1,13 @@
+# Run with Docker
+
+You can use Docker to run most Spaces locally.
+To view instructions to download and run Spaces' Docker images, click on the "Run with Docker" button on the top-right corner of your Space page:
+
+![Run with Docker button](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/spaces-run-with-docker.png)
+
+## Login to the Docker registry
+
+Some Spaces will require you to login to Hugging Face's Docker registry. To do so, you'll need to provide:
+- Your Hugging Face username as `username`
+- A User Access Token as `password`. Generate one [here](https://huggingface.co/settings/tokens).
+
```

Comment:
Should we also add some instructions about running with secrets? (e.g. specifying them in the docker run). Maybe this should be a UI change though